packages/fluentui/react-bindings/src/FocusZone/focusUtilities.ts

```diff
@@ -412,6 +414,8 @@ export function getWindow(rootElement?: Element | null): Window | undefined {
  * Helper to get the document object.
  *
  * @public
+ *
+ * @deprecated Use getDocument from @uifabric/utilities instead
  */
 export function getDocument(rootElement?: Element | null): Document | undefined {
```

Comment:
If signatures of these functions are compatible I suggest to reexport them instead.

Reply:
They are not entirely the same in all cases, but I think they are always compatible. They are also only used in `FocusZone` as far as I can tell so I __think__ it might be ok to do that.

Is my understanding correct in saying you're suggesting to delete these function implementations and just re-exporting them?

Reply:
> Is my understanding correct in saying you're suggesting to delete these function implementations and just re-exporting them?

Exactly 👍